lib/pbench/server/api/resources/query_apis/__init__.py

```diff
@@ -377,59 +392,57 @@ def __init__(
         """
         Base class constructor.
 
+        This method assumes and requires that a dataset will be located using
+        the controller and dataset name, so "controller" and "name" string-type
+        parameters must be defined in the subclass schema.
+
         Args:
             config: server configuration
             logger: logger object
             schema: API schema: for example,
                     Schema(
-                        Parameter("user", ParamType.USER, required=True),
-                        Parameter("start", ParamType.DATE)
+                        Parameter("controller", ParamType.STRING, required=True),
+                        Parameter("name", ParamType.STRING, required=True)
                     )
```

Comment:
I see that it says "for example", but having just `controller` and `name` doesn't seem viable -- it'll need at least one additional filed (e.g., `access`) -- so I suggest adding an another field or an ellipsis after line 405.

Reply:
In fact right now I have one other use in mind for this base class, which is DELETE; and it won't need any additional parameters. An ellipsis perhaps clarifies that you *can* but also could be read to imply that you *must*, which isn't true. I'll throw it in anyway if it makes you happy, since I won't be confused by it when I write delete. 😆